inspire_disambiguation/config.py

```diff
@@ -72,7 +72,6 @@
 
 instance_path = os.path.dirname(os.path.abspath(__file__))
 disambiguation_base_path = os.path.join(instance_path, "disambiguation")
-BASE_PATH = disambiguation_base_path
```

Comment:
why? shouldn't the environment override this? or is the issue that all the other variables after have already been instanciated so you need to override them explicitly too?

Reply:
Yes, this config file is parsed before env vars are loaded.

Reply:
what I'm saying is that this is useless, because the other paths are hardcoded here too.

Reply:
This is why I removed it?